prov/efa/src/efa_prov.c

```diff
@@ -91,6 +121,13 @@ static int efa_util_prov_initialize()
 			continue;
 		}
 
+		/* Set fabric name */
```

Comment:
You can still keep `efa_prov_info_set_fabric_attr` buit name `fabric_name` as an argument of the function?